### ISSUE TITLE
(Web Admin): Unable to add nodes - Can't locate object method "roles" via package

### ISSUE THREAD

**Issue by codeblueprint** [inverse-inc/packetfence#3434]
packetfence-release-1.2-7.el7.noarch.rpm (New installations)
https://PF_IPADDR:1443/admin/nodes#node/create
Attempting to add node either individually or in bulk results in the below error

> Error! An error occured while contacting the server. Please try again later.

Below error captured from journalctl

> Jul 31 18:51:38 PF_Host httpd_admin[2982]: httpd.admin(2050) ERROR: [mac:unknown] Caught exception in pfappserver::Controller::Node->create "Can't locate object method "roles" via package "pfappserver::Form::Node::Create::Import" at /usr/local/pf/lib/pfappserver/Form/Node/Create/Import.pm line 123." (pfappserver::PacketFence::Controller::Root::end)
> 
> Jul 31 18:51:38 PF_Host httpd_admin_access[2479]: 10.0.xx.xx - - [31/Jul/2018:18:51:38 -0400] "GET /node/create?_=1533077255518 HTTP/1.1" 500 2149 590 184059 "https://PF_Host:1443/admin/nodes" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36" "PF_Host:1443"

Tested with different servers.

![image](https://user-images.githubusercontent.com/212233/43492032-126c92ac-94f5-11e8-8552-44c68ed96473.png)

**Comment by nicholaspier** [comment]
#metoo

**Comment by jrouzierinverse** [comment]
Which version of PacketFence?

**Comment by codeblueprint** [comment]
Version 8.1.0
Released Tuesday, Jul 10, 2018 
using packetfence-release-1.2-7.el7.noarch.rpm on CentOS Linux release 7.5.1804 (Core)
@jrouzierinverse

**Comment by jrouzierinverse** [comment]
This has been fixed in devel by this commit 132918394c8062a452b5a843be5a81667636d8ce.
You can run ```/usr/local/pf/addons/pf-maint.pl``` to get the latest fixes.
You apply the patch by doing the following.

```
cd /usr/local/pf

curl https://github.com/inverse-inc/packetfence/commit/1329183.diff > 1329183.diff

patch -p1 1329183.diff

```
